### PR TITLE
Als007

### PR DESCRIPTION
**Pull request by aothms**
Only review commit https://github.com/buildingSMART/ifc-gherkin-rules/pull/78/commits/66b8b8eaacfee93f11341bdbad861ca7f11b9116 pelase

### CODE REVIEW

File: test/files/als005/fail-sectioned-solid-horizontal-wrong-identifier.ifc
Comment (Ghesselink) [inline test/files/als005/fail-sectioned-solid-horizontal-wrong-identifier.ifc:1]: add 'als005' to filename

`fail-als005-sectioned-solid-horizontal-wrong-identifier.ifc`

File: test/files/als006/pass-sectioned-solid-horizontal.ifc
Comment (Ghesselink) [inline test/files/als006/pass-sectioned-solid-horizontal.ifc:1]: add 'als006 to filename'

`pass-als006-sectioned-solid-horizontal.ifc`

File: test/files/als007/fail-sectioned-solid-horizontal-wrong-identifier.ifc
Comment (Ghesselink) [inline test/files/als007/fail-sectioned-solid-horizontal-wrong-identifier.ifc:1]: add 'als007' to filename

`fail-als007-sectioned-solid-horizontal-wrong-identifier.ifc`

File: test/files/als007/fail-sectioned-solid-horizontal-wrong-item.ifc
Comment (Ghesselink) [inline test/files/als007/fail-sectioned-solid-horizontal-wrong-item.ifc:1]: add 'als007' to filename

`fail-als007-sectioned-solid-horizontal-wrong-item.ifc`

File: test/files/als007/fail-sectioned-solid-horizontal-wrong-type.ifc
Comment (Ghesselink) [inline test/files/als007/fail-sectioned-solid-horizontal-wrong-type.ifc:1]: add 'als007' to filename

`fail-als007-sectioned-solid-horizontal-wrong-type.ifc`

File: test/files/als007/pass-sectioned-solid-horizontal.ifc
Comment (Ghesselink) [inline test/files/als007/pass-sectioned-solid-horizontal.ifc:1]: add 'als007' to filename

`pass-als007-sectioned-solid-horizontal.ifc`

**Review by Ghesselink — CHANGES_REQUESTED** [review]
Looks good. 
The rule appears to evaluate three conditions, each associated with a separate test file.

I apologize for the nitpicking. In the README.md is mentioned 

> Unit test files must follow this naming convention
> 
> Expected result-rule code-rule scenario-short_informative_description.ifc

The rule_code is not present. So for example 'pass-sectioned-solid-horizontal.ifc' becomes 'pass-**als005**-sectioned-solid-horizontal.ifc'

File: test/files/als007/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
+| fail-als007-sectioned-solid-horizontal-wrong-type.ifc       | fail              | ['On instance #20011=IfcShapeRepresenta...,(#79)) the following invalid value for RepresentationType has been found: Curve2D']                                                         |               |
+| pass-als007-sectioned-solid-horizontal.ifc                  | pass              | Rules passed                                                                                                                                                                           |               |
+| fail-als007-sectioned-solid-horizontal-wrong-identifier.ifc | fail              | ['On instance #20011=IfcShapeRepresenta...,(#79)) the following invalid value for RepresentationIdentifier has been found: FootPrint']                                                 |               |
+| fail-als007-sectioned-solid-horizontal-wrong-item.ifc       | fail              | ["The instance #20011=IfcShapeRepresentation(#1001,'Axis','Curve3D',(#54)) expected type 'IfcGradientCurve' for the attribute Items, but found [#54=IfcCompositeCurve((#...),.U.)]  "] |               |
```
Comment (pjanck) [inline test/files/als007/README.md:6]: Is there a certain order to the entries? I propose to always have them ordered "pass - fail".

**Review by pjanck — CHANGES_REQUESTED** [review]

File: test/files/als007/README.md (hunk 1 above)
Comment (Ghesselink) [reply test/files/als007/README.md:6]: That's perhaps clearer, indeed. The order is changed to the order you suggested in https://github.com/buildingSMART/ifc-gherkin-rules/pull/83

**Review by Ghesselink — APPROVED** [review]

**Comment by aothms** [comment]
Everything appears addressed. Thanks all.

**Pull request merged**